doc/Client-Portal Protocol.md

```diff
@@ -350,38 +356,44 @@ This a format for client reconnects.
      message: string(Message)
     }
 
-### 3.3.7 Participant Starts Publishing a WebRTC Stream to Room
+### 3.3.7 Participant Starts Publishing a Stream to Room
 **RequestName**: “publish”<br>
 
 **RequestData**: The PublicationRequest object with following definition:
 
+```
   object(PublicationRequest)::
     {
-     media: object(WebRTCMediaOptions),
-     attributes: object(ClientDefinedAttributes)
+     media: object(WebRTCMediaOptions) | null,
+     data: true | false,
+     transport: object(TransportOptions),
+     attributes: object(ClientDefinedAttributes) | null
     }
 
-    object(WebRTCMediaOptions)::
-      {
-       audio: {
-              source: "mic" | "screen-cast" | "raw-file" | "encoded-file"
-             }
-             | false,
-       video: {
-              source: "camera"| "screen-cast"  | "raw-file" | "encoded-file",
-              parameters:
-                {
-                 resolution: object(Resolution),
-                 framerate: number(FramerateFPS)
-                }
-             }
-             | false
-      }
+  object(WebRTCMediaOptions)::
+    {
+      audio: {
+            source: "mic" | "screen-cast" | "raw-file" | "encoded-file"
+            }
+            | false,
+      video: {
+            source: "camera"| "screen-cast"  | "raw-file" | "encoded-file",
+            parameters:
+              {
+                resolution: object(Resolution),
+                framerate: number(FramerateFPS)
+              }
+            }
+            | false
+    }
+```
+
 **ResponseData**: The PublicationResult object with following definition if **ResponseStatus** is “ok”:
 
   object(PublicationResult)::
     {
-     id: string(SessionId) //will be used as the stream id when it gets ready.
+      transportId: string(transportId),  // Can be reused in the following publication or subscription.
```

Comment:
I think this change is also for addTransceiver on single connection, how about put this in another PR?

Reply:
Adding `transportId` may not enough for reuse PeerConnection for multiple RTP channels. This PR only adds it for QUIC connections. You may need more changes to make it possible for combining PeerConnections. And `addTransceiver` at client side doesn't reuse PeerConnection in early stage.